runtime/registry.go

```diff
@@ -556,6 +557,9 @@ func (r *registryCache) updateProjectConfig(iwc *instanceWithController) error {
 
 	rillYAML, err := rillv1.ParseRillYAML(iwc.ctx, repo, iwc.instanceID)
 	if err != nil {
+		if strings.Contains(err.Error(), "rill.yaml not found") { // empty project
+			return nil
+		}
```

Comment:
Since this is in the same language/repo, it's better to declare `rillv1.ErrRillYAMLNotFound` and use `errors.Is` to check here